LunaMary/src/QuizLibrary.java

```diff
@@ -0,0 +1,40 @@
+import java.util.List;
+
+public class QuizLibrary {
+
+
+    private List<Quiz> quizList;
+    private List<AnswerPerson> answerPeople;
+    private String type;
```

Comment:
The library shouldn't have an attribute type.
Instead you could have a list of types that you have Quizzes for.